Dockerfile

```diff
@@ -10,8 +10,10 @@ RUN dnf install -y gcc git make xz-devel
 
 WORKDIR /tmp
 
-RUN git clone --depth 1 --branch v1.21.1 https://github.com/ipxe/ipxe.git && \
-      cd ipxe/src && \
+RUN git clone https://github.com/ipxe/ipxe.git && \
+      cd ipxe && \
+      git reset --hard 9062544f6a0c69c249b90d21a08d05518aafc2ec && \
```

Comment:
How did you pick this version? It does not seem to be the latest. Let's leave a comment on what this version corresponds to (date, reason of picking).

Also, maybe it's time we introduce an argument for the iPXE version so that the users can revert more easily (and we can test new versions easier)?

Reply:
It's not the latest as unfortunately the latest does not work :/
Ideally I'm trying to take the update with small steps at a time, but I'll provide a better commit message with some of the most important changes.
Also good idea about the argument, I'll work on that too